build.gradle

```diff
@@ -87,6 +88,10 @@ dependencies {
   testImplementation 'com.github.stefanbirkner:system-rules:1.18.0'
   testImplementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
   testImplementation "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+
+  testCompileOnly gradleTestKit()
```

Comment:
Why do we need this? Should not this maybe be `integrationTestCompileOnly`?

Reply:
This is required to have [`GradleTestRunner`](https://github.com/vanniktech/gradle-maven-publish-plugin/pull/49/files#diff-b9585b580f4e536ac258791d99aa04ceR4) available in tests to execute various Gradle commands in tests.
I tried initially the `integrationTestCompileOnly` too but for some unknown to me reason classes Gradle test kit were not available in integration test I've created 👀.

Reply:
@vanniktech ok, it works (obviously). I can't recall already why `import org.gradle.testkit.runner.GradleRunner` wasn't working for custom integration test source set and I had to change from `integrationTestCompileOnly` to `testCompileOnly`.